packages/auth/src/accessToken/routes.ts

```diff
@@ -164,7 +164,7 @@ async function rotateToken(
     const errorMessage =
       error instanceof Error ? error.message : 'Could not rotate token'
     throw new GNAPServerRouteError(
-      400,
+      404,
```

Comment:
Do we throw in the other service methods that may lead to this 404? or do we just want this error message for the `if (!newToken)` case?

Reply:
CC @njlie